docs/modules/operation/pages/events/sources/syslog.adoc

```diff
@@ -4,30 +4,39 @@
 
 Syslog messages sent over the network to {page-component-title} can be transformed into events according to preconfigured rules.
 
-IMPORTANT: The `Syslogd` service daemon, which enables {page-component-title} to receive syslog messages over the network, must be enabled for this functionality to work.
-This service daemon is *disabled* by default.
+IMPORTANT: The Syslogd service daemon, which enables {page-component-title} to receive syslog messages over the network, must be enabled for this functionality to work.
+This service daemon is disabled by default.
 
 == Parsers
 
 Different parsers can be used to convert the syslog message fields into {page-component-title} event fields.
 
 [options="header, %autowidth"]
 |===
```

Comment:
This table isn't rendering properly. The Description field is under the parser.

Reply:
doesn't have `cols` set.